.github/workflows/push.yml

```diff
@@ -0,0 +1,41 @@
+on: push
+name: Build and Publish
+jobs:
+  dockerLint:
+    name: Docker Lint
```

Comment:
Calling these `Docker Lint` when this is also the name of one of the discreet steps seems a little confusing to me. Maybe call this just `Docker` or something? Not sure. It will still all work from what I can tell.